src/style.css

```diff
@@ -1,3 +1,78 @@
+html {
+  min-width: 1024px;
+  max-width: 1200px;
+}
+
 body {
+  background-color: #e5e5e5;
+  margin: 0;
+  padding: 0;
+  font-family: Roboo, sans-serif;
+}
+
+
+.img {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin: 10px 0 10px 50px;
+}
+
+
+.header {
+  display: flex;
+  padding: 0 40px 0 50px;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+}
+
+.nav {
+  display: inline-block;
+  height: 60px;
+}
+
+.nav__list {
+  display: flex;
   margin: 0;
+  height: 100px;
+  list-style: none;
+}
+
+.nav__item {
+  position: relative;
+  padding-right: 20px;
+  font-style: normal;
+  font-weight: 500px;
+  line-height: 14px;
+  font-size: 12px;
+  align-items: center;
+  text-transform: uppercase;
+  display: inline-block;
+}
+
+.nav__link {
+  text-decoration: none;
+  padding: 23px 0;
+  color: #000;
+  cursor: pointer;
+}
+
+.is_active {
+  border-bottom: 4px solid #00acdc;
```

Comment:
Move all repeatable colors to the variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)